src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -112,8 +119,11 @@ case class SingleWorkflowRunnerActor(source: WorkflowSourceFiles,
       issueReply using updatedData
   }
 
+  var isDone = false
```

Comment:
Is there a way to synchronously retrieve the current state? I couldn't find one but I'm not so familiar with Actors.